keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -295,6 +295,7 @@ Consider including folks who also work outside the SIG or subproject.
 | Risk | Impact | Mitigation |
 | --- | --- | --- |
 | Confusing users with a new config file | Low | Documentation and education |
+| Users trust the kuberc is being used | High | Provide mechanism warning users default kuberc is not being picked up |
```

Comment:
Do we need to remove this?.

Reply:
We do, that was added with that story 5 removed. Good catch, thank you.